### PR TITLE
Update addCommand function

### PR DESCRIPTION
**Pull request by Xuyan0518**
Add function now supports batch processing for addition of multiple categories and descriptions at one time.

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/AddCommandParser.java [hunk 1]
```diff
@@ -52,47 +53,17 @@ && arePrefixesPresent(argMultimap, PREFIX_DESCRIPTION)) {
...
-            }
-            if (argMultimap.getValue(PREFIX_DESCRIPTION).isPresent()) {
-                String descriptionString = argMultimap.getValue(PREFIX_DESCRIPTION).get().trim();
-                if (!descriptionString.isEmpty()) {
-                    description = descriptionString.split(",");
-                } else {
-                    throw new ParseException(AddCategoryCommand.ENTRY_NOT_ADDED);
-                }
+            List<String> categories = argMultimap.getAllValues(PREFIX_CATEGORY);
+            List<String> descriptions = argMultimap.getAllValues(PREFIX_DESCRIPTION);
```
Comment (NusMinato) [inline src/main/java/seedu/address/logic/parser/AddCommandParser.java:57]: Nice implementation of the add command batch processing.

**Review by NusMinato — COMMENTED** [review]
Good job! Nice implementation of the add command batch processing.

**Review by Cedricaca — COMMENTED** [review]
nice changes

File: src/main/java/seedu/address/logic/parser/AddCommandParser.java [hunk 2]
```diff
@@ -52,47 +53,17 @@ && arePrefixesPresent(argMultimap, PREFIX_DESCRIPTION)) {
...
-                }
-            }
-            if (argMultimap.getValue(PREFIX_DESCRIPTION).isPresent()) {
-                String descriptionString = argMultimap.getValue(PREFIX_DESCRIPTION).get().trim();
-                if (!descriptionString.isEmpty()) {
-                    description = descriptionString.split(",");
-                } else {
-                    throw new ParseException(AddCategoryCommand.ENTRY_NOT_ADDED);
-                }
+            List<String> categories = argMultimap.getAllValues(PREFIX_CATEGORY);
```
Comment (HongRay) [inline src/main/java/seedu/address/logic/parser/AddCommandParser.java:56]: Great use of `List<String> categories = argMultimap.getAllValues(PREFIX_CATEGORY);` I use it too!

**Pull request merged**